suite/tests/api/ir_aarch64.h

```diff
@@ -127,6 +127,15 @@ test_instr_encoding(void *dc, uint opcode, instr_t *instr, const char *expected)
     return result;
 }
 
+const reg_id_t Xn_six_offset_0[6] = { DR_REG_X0,  DR_REG_X5,  DR_REG_X10,
+                                      DR_REG_X15, DR_REG_X20, DR_REG_X30 };
+const reg_id_t Xn_six_offset_1[6] = { DR_REG_X0,  DR_REG_X6,  DR_REG_X11,
+                                      DR_REG_X17, DR_REG_X22, DR_REG_X30 };
+const reg_id_t Xn_six_offset_2[6] = { DR_REG_X0,  DR_REG_X7,  DR_REG_X12,
```

Comment:
Xn_six_offset_3 and Xn_six_offset_2 seem unused.

Reply:
Oops! Deleted.